packages/vkui/src/components/Accordion/Accordion.tsx

```diff
@@ -1,8 +1,10 @@
-import React, { HTMLProps } from 'react';
+import React, { DetailsHTMLAttributes } from 'react';
+import { classNames } from '@vkontakte/vkjs';
 import { HasRef } from '../../types';
 import { AccordionSummary } from './AccordionSummary';
+import styles from './Accordion.module.css';
 
-export type AccordionProps = HTMLProps<HTMLDetailsElement> & HasRef<HTMLDetailsElement>;
+export type AccordionProps = DetailsHTMLAttributes<HTMLDetailsElement> & HasRef<HTMLDetailsElement>;
```

Comment:
```suggestion
export type AccordionProps = React.DetailsHTMLAttributes<HTMLDetailsElement> & HasRef<HTMLDetailsElement>;
```

См. мой предыдущий коммент. (%